packages/commonwealth/client/scripts/views/pages/CommunityGroupsAndMembers/Members/GroupsSection/GroupCard/GroupCard.tsx

```diff
@@ -71,6 +82,58 @@ const GroupCard = ({
         <RequirementCard key={index} {...r} />
       ))}
 
+      {allowLists && (
+        <>
+          <CWText type="h5">Allow List</CWText>
+          <CWText type="b2">
+            These users are added directly to the group and may bypass
+            additional requirements
+          </CWText>
+          <div className="allowlist-table">
+            <table>
```

Comment:
we should be using CWTable component

Reply:
just realised this is this view, so simple flex or grid would do the work 

![image](https://github.com/hicommonwealth/commonwealth/assets/14819225/024c651d-53e2-40bd-831d-f9adfad0a05a)

Reply:
The problem here is that I think CWTable is too opinionated. If you check the figma:
https://www.figma.com/file/u9xVULF2IzBU4riycghQil/Gating-v2?type=design&node-id=522-20776&mode=design&t=IWgV0xRwiVirZq0f-0

We have:
<img width="479" alt="image" src="https://github.com/hicommonwealth/commonwealth/assets/14794654/0ea92fdf-70b4-4e7d-8b55-85aa90954075">

But notice that we don't have any horizontal lines between the rows, or the sortable header on a different colored background. If I convert to CWTable, will it be possible to remove these potions? Or will I need to refactor CWTable itself to support it?

Reply:
I was misled but <table> tag you used. This is not a case for CWTable, rather flexbox or grid as I mentioned above